<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findpassword</class>
 <widget class="QWidget" name="findpassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>350</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>450</width>
     <height>350</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame{
/*frame是整个widget里面的容器，用于放置控件 整体拖动*/
/*背景图片*/
background-image: url(:/new/prefix1/findPassword.png);
/*边缘圆角半径设置*/
border: 1px solid rgba(0,191,255, 255);
border-radius:5px; 
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="close">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">/*关闭按钮*/</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="SourceFile.qrc">:/new/prefix1/close.png</pixmap>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>9</y>
      <width>391</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>385</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="brandName_3">
           <property name="font">
            <font>
             <family>方正姚体</family>
             <pointsize>18</pointsize>
             <weight>50</weight>
             <italic>true</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">/*产品名称*/
color:#DCDCDC;/*设置字体颜色*/</string>
           </property>
           <property name="text">
            <string>账号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="brandName_2">
           <property name="font">
            <font>
             <family>方正姚体</family>
             <pointsize>18</pointsize>
             <weight>50</weight>
             <italic>true</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">/*产品名称*/
color:#DCDCDC;/*设置字体颜色*/</string>
           </property>
           <property name="text">
            <string>密保一</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="brandName_4">
           <property name="font">
            <font>
             <family>方正姚体</family>
             <pointsize>18</pointsize>
             <weight>50</weight>
             <italic>true</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">/*产品名称*/
color:#DCDCDC;/*设置字体颜色*/</string>
           </property>
           <property name="text">
            <string>密保二</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="brandName_5">
           <property name="font">
            <font>
             <family>方正姚体</family>
             <pointsize>18</pointsize>
             <weight>50</weight>
             <italic>true</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">/*产品名称*/
color:#DCDCDC;/*设置字体颜色*/</string>
           </property>
           <property name="text">
            <string>密保三</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="accout_line">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>15</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"> /*文本输入框 账号输入*/
QLineEdit{
/*非悬浮时颜色设置*/
color:#FFFAFA;
border-style:outset;/*设置边框*/
background-color: rgba(255,255,255,0);
border-radius:2px;/*边框圆角半径设置*/
border: 1px solid rgba(70,130,180, 80)

}
QLineEdit::hover
{
/*悬浮时颜色设置*/
background-color:rgba(135,206,250, 255);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>25</number>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordProtect1">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>15</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"> /*文本输入框 账号输入*/
QLineEdit{
/*非悬浮时颜色设置*/
color:#FFFAFA;
border-style:outset;/*设置边框*/
background-color: rgba(255,255,255,0);
border-radius:2px;/*边框圆角半径设置*/
border: 1px solid rgba(70,130,180, 80)

}
QLineEdit::hover
{
/*悬浮时颜色设置*/
background-color:rgba(135,206,250, 255);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>25</number>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordProtect2">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>15</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"> /*文本输入框 账号输入*/
QLineEdit{
/*非悬浮时颜色设置*/
color:#FFFAFA;
border-style:outset;/*设置边框*/
background-color: rgba(255,255,255,0);
border-radius:2px;/*边框圆角半径设置*/
border: 1px solid rgba(70,130,180, 80)

}
QLineEdit::hover
{
/*悬浮时颜色设置*/
background-color:rgba(135,206,250, 255);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>25</number>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordProtect3">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>15</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"> /*文本输入框 账号输入*/
QLineEdit{
/*非悬浮时颜色设置*/
color:#FFFAFA;
border-style:outset;/*设置边框*/
background-color: rgba(255,255,255,0);
border-radius:2px;/*边框圆角半径设置*/
border: 1px solid rgba(70,130,180, 80)

}
QLineEdit::hover
{
/*悬浮时颜色设置*/
background-color:rgba(135,206,250, 255);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>25</number>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="brandName">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>131</width>
        <height>49</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>方正姚体</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">/*产品名称*/
color:#DCDCDC;/*设置字体颜色*/</string>
      </property>
      <property name="text">
       <string>找回密码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="commit_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>290</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>方正姚体</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">/*登录按钮*/
QPushButton{
/*静置时设置*/
color:#FFFAFA;/*字体颜色设置*/
background-color:rgba(70,130,180, 255);/*背景颜色设置*/;
border-style:inset;/*背景颜色设置*/
border-radius:2px;/*边框圆角半径设置*/
border: 1px solid rgba(70,130,180, 255);
}

QPushButton:hover{
/*悬停时设置*/ /*由于与内部代码有冲突 已弃用*/
background-color:rgba(135,206,250, 255);
border-style:inset;/*边框设置*/
}
QPushButton:pressed{
/*按压时设置*/
background-color:rgba(173,216,230, 255);
/*背景颜色设置*/
border-style:inset;/*边框设置*/
}</string>
      </property>
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="SourceFile.qrc"/>
 </resources>
 <connections/>
</ui>
